CONTRIBUTING.md

```diff
@@ -34,18 +34,38 @@ accept your pull requests.
 ### More Information on Forks and Pull Requests
 
 If you are just getting started with `git` and [GitHub](https://github.com),
-we think the following links may help you understand the usual workflow used
-in this project:
-
-* [Forking Projects](https://guides.github.com/activities/forking/)
-* [Understanding the GitHub Flow](https://guides.github.com/introduction/flow/)
-* [Syncing a Fork](https://help.github.com/articles/syncing-a-fork/)
-  * A more succinct guide to [Keeping a Fork up to date](https://gist.github.com/CristinaSolana/1885435)
-
-If your pull request has a conflict with the destination branch we request
-that you [rebase](https://help.github.com/articles/about-git-rebase/) your
-branch against the destination branch. Reviewing PRs that include both existing
-and new changes is a source of error that we would rather eliminate.
+this section might be helpful. In this project we use the (more or less)
+standard [GitHub workflow][workflow-link]:
+
+1. You create a [fork][fork-link] of [google-cloud-cpp][repo-link]
+2. You pick an existing [GitHub bug][mastering-issues] to work on.
+3. You start a new [branch][about-branches] for each feature (or bug fix).
+4. You submit a [pull-request][about-pull-requests] to merge your branch into
+   `GoogleCloudPlatform/google-cloud-cpp`.
+5. Your reviewers may ask questions, suggest improvements or alternatives. You
+   address those by either answering the questions in the review or adding more
+   [commits][about-commits] to your branch.
+6. From time to time your pull request may have conflicts with the destination
+   branch, if so, we request that you [rebase][about-rebase] your branch
+   instead of merging with the destination branch. The reviews can become very
+   confusing if you merge during a pull request.
+7. If one of the CI builds fail please see below, most of the CI builds can
+   be reproduced locally on your workstations using Docker.
+8. Eventually the reviewers accept your changes, and they are merged into the
+   `master` branch.
+9. Periodically, and almost certainly before starting your next task, you need
```

Comment:
This is nice, but it links to a gist, which can go away at any moment. That is the right workflow; do you want to add it here?

Reply:
Done. And since we have the simpler workflow here I pointed to the GitHub documentation instead.